pyproject.toml

```
@@ -26,7 +26,7 @@ dependencies = [
     "cachetools>=5.0.0,<6.0",
     "defusedxml>=0.7.1",
     "numpy>=1.23.2,<3.0",
-    "pillow>=10.0.1,<11.0",
+    "pillow>=11,<11.1",
```

```suggestion
    "pillow>=10.0.1",
```
@edenhaus Support for pillow `>11.0` is required to build wheels for Python 3.13 on some platforms with Home Assistant. See https://github.com/home-assistant/core/pull/130194

https://pillow.readthedocs.io/en/stable/releasenotes/11.0.0.html#python-3-13